metricbeat/include/list.go

```diff
@@ -85,6 +85,10 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/system/network"
 	_ "github.com/elastic/beats/metricbeat/module/system/process"
 	_ "github.com/elastic/beats/metricbeat/module/system/socket"
+	_ "github.com/elastic/beats/metricbeat/module/vsphere"
+	_ "github.com/elastic/beats/metricbeat/module/vsphere/datastore_usage"
```

Comment:
Do we need the appendix `usage` for the metricset names? Are there potential other metricsets for `datastore` for example?